en/docs/developer-portal/manage-application.md

```diff
@@ -8,46 +8,150 @@ Let's get started!
+2. Click **Applications** and click **+Create**.
+3. Enter the application name and select the usage policy. You can optionally also add the application description. Click **Create**.
 4. You are redirected to the application overview page. You can view the throttling tier, the token type, workflow status, and the application owner of the API. 
 
 ## Generate Keys
 
-Choreo provides an OAuth 2.0 bearer token-based authentication for API access. An API access token/key is a string, passed as an HTTP header of an API request to authenticate the API access. 
+Choreo provides an OAuth 2.0 bearer token-based authentication for API access. An API access token/key is a string that is used to authenticate the API access and is passed as an HTTP header of an API request. 
 
-Once you create an application in Choreo, you can then generate the credentials for the application. When you generate the credentials for the first time, Choreo gives you a consumer key and consumer secret for the application. The consumer key becomes the application's unique identifier, similar to a user's user name, and is used to authenticate the application or user. You can use this consumer key and consumer secret to generate an API access token by invoking the token endpoint. You can also revoke the access token by invoking the revoke endpoint. For testing purposes, you can generate a test token via the UI. However, we strongly recommend not to use the test token in your production environment.
+Once you create an application in Choreo, you can generate the credentials for the application. You will receive a consumer key and consumer secret when you generate credentials. The consumer key becomes the application's unique identifier, similar to a user's username, and is used to authenticate the application or user. You can use the consumer key and consumer secret to generate an API access token by invoking the token endpoint. You can also revoke the access token by invoking the revoke endpoint. For testing purposes, you can generate a test token via the UI. However, we strongly recommend not to use the test token in your production environment.
 
 This section will walk you through the steps to generating an API access token in Choreo. 
 
 Let's get started!
 
-1. Sign in to Choreo Developer Portal at [https://devportal.choreo.dev](https://devportal.choreo.dev). Alternatively, click the Developer Portal link on the Choreo console header. 
+### Access Token for Production 
 
-    ![Developer Portal](../assets/img/developer-portal/manage-applications/developer-portal.png){.cInlineImage-half}
+Depending on your use case, you can generate an access token using one of the two grant types available in Choreo: Client Credentials and Token Exchange. 
+
+!!! note
+    We recommend you use the **Client Credentials** grant type to obtain an access token **for testing purposes** and use the **Token Exchange** grant type **for production use**. 
```

Comment:
We recommend you use the **Token Exchange** grant type **for production use** and **Client Credentials** grant type to obtain an access token **for testing purposes**.?